tfjs-backend-webgpu/src/binary_op_webgpu.ts

```diff
@@ -70,13 +70,11 @@ export class BinaryOpProgram implements WebGPUProgram {
         this.variableComponents = [4, 4];
         this.type = 'vec4';
       } else if (
-          (op === BinaryOpType.SUB || op === BinaryOpType.ADD ||
-           op === BinaryOpType.MUL || op === BinaryOpType.DIV) &&
```

Comment:
Since we have forced to convert `float` to `vec4` in shader, there is no ops limitations anymore.